driver/esp32/ili9XXX.py

```diff
@@ -576,3 +588,104 @@ def __init__(self,
+            {'cmd': 0x84, 'data': bytes([0x40])},
+            {'cmd': 0x85, 'data': bytes([0xFF])},
+            {'cmd': 0x86, 'data': bytes([0xFF])},
+            {'cmd': 0x87, 'data': bytes([0xFF])},
+            {'cmd': 0x88, 'data': bytes([0x0A])},
+            {'cmd': 0x89, 'data': bytes([0x21])},
+            {'cmd': 0x8A, 'data': bytes([0x00])},
+            {'cmd': 0x8B, 'data': bytes([0x80])},
+            {'cmd': 0x8C, 'data': bytes([0x01])},
+            {'cmd': 0x8D, 'data': bytes([0x01])},
+            {'cmd': 0x8E, 'data': bytes([0xFF])},
+            {'cmd': 0x8F, 'data': bytes([0xFF])},
+            {'cmd': 0xB6, 'data': bytes([0x00, 0x00])}, 
+            {'cmd': 0x36, 'data': bytes([self.rotation | self.colormode])},
+            {'cmd': 0x3A, 'data': bytes([0x05])},
+            {'cmd': 0x90, 'data': bytes([0x08, 0x08, 0x08, 0x08])},
+            {'cmd': 0xBD, 'data': bytes([0x06])},
+            {'cmd': 0xBC, 'data': bytes([0x00])},
+            {'cmd': 0xFF, 'data': bytes([0x60, 0x01, 0x04])},
+            {'cmd': 0xC3, 'data': bytes([0x13])},
+            {'cmd': 0xC4, 'data': bytes([0x13])},
+            {'cmd': 0xC9, 'data': bytes([0x22])},
+            {'cmd': 0xBE, 'data': bytes([0x11])},
+            {'cmd': 0xE1, 'data': bytes([0x10, 0x0E])},
+            {'cmd': 0xDF, 'data': bytes([0x21, 0x0c, 0x02])},
+            {'cmd': 0xF0, 'data': bytes([0x45, 0x09, 0x08, 0x08, 0x26, 0x2A])},
+            {'cmd': 0xF1, 'data': bytes([0x43, 0x70, 0x72, 0x36, 0x37, 0x6F])},
+            {'cmd': 0xF2, 'data': bytes([0x45, 0x09, 0x08, 0x08, 0x26, 0x2A])},
+            {'cmd': 0xF3, 'data': bytes([0x43, 0x70, 0x72, 0x36, 0x37, 0x6F])},
+            {'cmd': 0xED, 'data': bytes([0x1B, 0x0B])},
+            {'cmd': 0xAE, 'data': bytes([0x77])},
+            {'cmd': 0xCD, 'data': bytes([0x63])},
+            {'cmd': 0x70, 'data': bytes([0x07, 0x07, 0x04, 0x0E, 0x0F, 0x09, 0x07, 0x08, 0x03])},
+            {'cmd': 0xE8, 'data': bytes([0x34])},
+            {'cmd': 0x62, 'data': bytes([0x18, 0x0D, 0x71, 0xED, 0x70, 0x70, 0x18, 0x0F, 0x71, 0xEF, 0x70, 0x70])},
+            {'cmd': 0x63, 'data': bytes([0x18, 0x11, 0x71, 0xF1, 0x70, 0x70, 0x18, 0x13, 0x71, 0xF3, 0x70, 0x70])},
+            {'cmd': 0x64, 'data': bytes([0x28, 0x29, 0xF1, 0x01, 0xF1, 0x00, 0x07])},
+            {'cmd': 0x66, 'data': bytes([0x3C, 0x00, 0xCD, 0x67, 0x45, 0x45, 0x10, 0x00, 0x00, 0x00])},
+            {'cmd': 0x67, 'data': bytes([0x00, 0x3C, 0x00, 0x00, 0x00, 0x01, 0x54, 0x10, 0x32, 0x98])},
+            {'cmd': 0x74, 'data': bytes([0x10, 0x85, 0x80, 0x00, 0x00, 0x4E, 0x00])},
+            {'cmd': 0x98, 'data': bytes([0x3e, 0x07])},
+            {'cmd': 0x35, 'data': bytes([0])},
+            {'cmd': 0x21, 'data': bytes([0])},
+            {'cmd': 0x11, 'data': bytes([0]), 'delay': 20},
+            {'cmd': 0x29, 'data': bytes([0]), 'delay': 120}
+        ]
+        
+        super().__init__(miso, mosi, clk, cs, dc, rst, power, backlight, backlight_on, power_on,
```

Comment:
Just in case we ever change any of the argument names, I recommend using named arguments for all arguments like ili9341 does:

https://github.com/lvgl/lv_binding_micropython/blob/dadad31bdce92fc96ffac6b2805ac0d5d4b884eb/driver/esp32/ili9XXX.py#L486-L491

Reply:
I am not sure what you mean with this one, all of the arguments in the function init are keyword parameters, the same as for the ili9341 code. If you are referring to the super().__init__ call then the ili9341 code is the same as mine